.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+*/.next/*
+.next/*
+package-lock.json
```

Comment:
please keep this in the project, it helps that everyone installs the exact same versions of the packages